ci/kubetest/test_clickhouse_service_loadbalancer.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from utils import cleanup_k8s, get_clickhouse_cluster_service_spec, helm_install, wait_for_pods_to_be_ready
+
+HELM_INSTALL_CMD = """
+helm upgrade \
+    --install \
+    -f ../../ci/values/kubetest/test_clickhouse_service_loadbalancer.yaml \
+    --timeout 30m \
+    --create-namespace \
+    --namespace posthog \
+    posthog ../../charts/posthog \
+    --wait-for-jobs \
+    --wait
+"""
+
+
+@pytest.fixture
+def setup(kube):
+    cleanup_k8s()
+    helm_install(HELM_INSTALL_CMD)
+    wait_for_pods_to_be_ready(kube)
+
+
+def test_helm_install(setup, kube):
```

Comment:
Q: is this just boilerplate, safe to ignore?

Reply:
Yep, it tests that helm installs cleanly